site/src/selector.rs

```diff
@@ -299,13 +325,13 @@ impl StatisticSeries {
             .get_pstats(&sids, &aids)
             .await
             .into_iter()
-            .zip(&statistic_descriptions)
+            .zip(statistic_descriptions)
             .filter(|(points, _)| points.iter().any(|value| value.is_some()))
-            .map(|(points, (&(benchmark, profile, scenario, metric), _))| {
+            .map(|(points, (test_case, metric, _))| {
                 SeriesResponse {
                     series: StatisticSeries {
                         artifact_ids: ArtifactIdIter::new(artifact_ids.clone()),
-                        points: if metric == *"cpu-clock" || metric == *"task-clock" {
+                        points: if *metric == *"cpu-clock" || *metric == *"task-clock" {
```

Comment:
Could we just normalize these values when they are gathered (and then run a query on production DB that will normalize the existing values in the DB), and then get rid of this annoying special case?

Reply:
I'd be fine with that - I think the DB query might be a little annoying depending on how many of these there are but probably okay.